README.md

```diff
@@ -45,12 +46,19 @@ The [Documentation Contributor Guide](https://strimzi.io/contributing/guide/) de
 
 If you want to get in touch with us first before contributing, you can use:
 
-- [Strimzi mailing list](https://www.redhat.com/mailman/listinfo/strimzi)
-- [Strimzi Slack workspace](https://join.slack.com/t/strimzi/shared_invite/enQtMzU2Mjk3NTgxMzE5LTYyMTUwMGNlMDQwMzBhOGI4YmY4MjhiMDgyNjA5OTk2MTFiYjc4M2Q3NGU1YTFjOWRiMzM2NGMwNDUwMjBlNDY)
+- [#strimzi channel on CNCF Slack](https://slack.cncf.io/)
```

Comment:
ditto as above

Reply:
That links is useless unless you are already registered. So I think we should keep what I have there.